package.json

```diff
@@ -4,7 +4,7 @@
   "version": "0.0.1",
   "type": "module",
   "scripts": {
-    "dev": "vite",
+    "dev": "ionic serve --host=127.0.0.1 --port=8100 --no-open",
```

Comment:
I actually see that as a feature and not a bug. Since `nmdc-server` requires redirect origins to be specifically enumerated in an [allow list](https://github.com/microbiomedata/nmdc-server/blob/5e7c6e04173d8474e86d7607d42097a50f476af7/nmdc_server/auth.py#L224), you wouldn't be able to log in if you ended up running on port `8101` without going in and futzing around with `nmdc-server` configuration.

Reply:
This is, however, making me realize that maybe I should add something to the `README` about what steps you need to take on the `nmdc-server` side of things if you want to work with both projects together locally.

Reply:
Ah, OK, that makes sense. I hadn't thought about that.

Reply:
Added a couple new sections to `README.md` about connecting to either a local or the dev `nmdc-server`.